geopy/geocoders/arcgis.py

```diff
@@ -109,7 +109,7 @@ def _authenticated_call_geocoder(self, url, timeout=None):
         )
         return self._base_call_geocoder(request, timeout=timeout)
 
-    def geocode(self, query, exactly_one=True, timeout=None):
+    def geocode(self, query, exactly_one=True, out_fields=None, timeout=None):
```

Comment:
Could you please put the `out_fields` argument after `timeout`? Otherwise it would break current usages of `geocode` call, where timeout is passed as a positional arg. Also the base method has `timeout` too, so we should stick to that order.